package.json

```diff
@@ -46,12 +47,15 @@
     "prettier": "^1.16.1"
   },
   "scripts": {
+    "applyIdsToChartDefinitions": "node scripts/chartsDefinitionsApplyUniqueId.js",
```

Comment:
`scrips` folder should also be added to `lint-staged`

Reply:
scrips ?